astropy/table/operations.py

```diff
@@ -331,7 +331,7 @@ def hstack(tables, join_type='outer',
     return out
 
 
-def unique(input_table, keys=None, silent=False):
+def unique(input_table, keys=None, keep='first', silent=False):
```

Comment:
Put `keep` as the last keyword for back compatibility.  We need to allow for someone writing `unique(table, 'col0', True)` in their code based on the astropy 1.1 API.